.dockerignore

```diff
@@ -0,0 +1,31 @@
+# See https://docs.docker.com/engine/reference/builder/#dockerignore-file for more about ignoring files.
```

Comment:
- [ ] The mobile view has not been implemented yet.  I understand this is a work in progress or a future update.


_tip:  If you feel you've completed 90% of the features, kindly say in your PR description what features will be left out to give a heads-up to the final evaluator and reviewers._